src/CitMovie.Api/Controller/BookmarkController.cs

```diff
@@ -0,0 +1,76 @@
+using CitMovie.Business.Managers;
+using CitMovie.Models.DTOs;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+namespace CitMovie.Api.Controllers
+{
+    [ApiController]
+    [Route("api/bookmarks")]
+    [Authorize(Policy = "user_scope")]
+    public class BookmarkController : ControllerBase
+    {
+        private readonly IBookmarkManager _bookmarkManager;
+
+        public BookmarkController(IBookmarkManager bookmarkManager) =>
+            _bookmarkManager = bookmarkManager;
+
+        private int GetUserId() =>
+            int.Parse(User.FindFirstValue("user_id") ?? throw new UnauthorizedAccessException("User ID not found"));
+
```

Comment:
Please make use of the user_scope policy to check if user is allowed to use an endpoint, see follow or user controller for example of how to implement